### ISSUE TITLE
Colors of popup are not clearly visible in High Contrast 'Aquatic' and 'Desert' modes.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8506]
**What is the current behavior? Describe the bug**
Colors of popup are not clearly visible in High Contrast 'Aquatic' and 'Desert' modes.
**Actual Results 1:**
Colors are not clearly visible in Border style and Background 'colors picker' popup.

**Actual Results 2:**
color in the slider is not visible in HC modes.

**Actual Result 3:**
colors are not clearly visible in HC mode.

**Pre-requisites:** Turn on High Contrast (Aquatic/Desert) using Settings->Accessibility->Contrast Themes->Aquatic/Desert.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**

**Repro steps for issue 1**
Step 1: Please open https://codepen.io/keyapanja/pen/JjBMdBg in Edge browser.
Step 2: The default focus would be on HTML body
Step 3: Press ESC, the focus would be on 'Open HTML settings'
Step 4: Navigate to Codepen preview Area.
Step 5: Navigate to Formatting toolbar
Step 6:Navigate to table and Insert table.
Step 7 :Navigate to 'Table' in toolbar and navigate to 'Table properties' and invoke it.
Step 8:In Table properties dialog, invoke advanced tab.
Step 9:Invoke Border style and Background color buttons, 'colors picker' popup will appear.
Step 10:Observe the 1st issues

**Repro steps for issue 2**
Step 1:Invoke custom color from the color pop-up in Border style and Background color color picker.
Step 2:Observe the 2nd issue.

**Repro steps for issue 3**
Step 1:Navigate to Format menu item in formatting toolbar.
Step 2: Invoke 'format' menu item. 
Step 3:invoke 'Text color' and 'Background color' option menu
Step 4:color picker popup will appear 
Step 5:Observe the issue.

**What is the expected behavior?**
Colors of popup should be clearly visible in High Contrast 'Aquatic' and 'Desert' modes.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
OS Version: Windows 11 Dev 22H2(OS Build 22621.1265 )
Edge Dev Browser Version: 110.0.1587.57 (Official build) (64-bit)
Tiny MCE version V6.0

![colors in popup_normal mode](https://user-images.githubusercontent.com/93735775/222409641-575ef635-8942-4ca0-929d-d6d03172250d.png)
![colors in custom colors_HC modes](https://user-images.githubusercontent.com/93735775/222409649-d15c98eb-750d-4942-a053-e846694413ba.png)
![colors in custom colors_normal mode](https://user-images.githubusercontent.com/93735775/222409652-3638c3bc-7621-48c8-ac30-5625a2d379c1.png)
![colors in popup_HC modes](https://user-images.githubusercontent.com/93735775/222409654-0b5ac4d5-e9b2-4c6c-86ae-3871b95d850b.png)

**Comment by msftedad** [comment]
Hi Team,

Any update on this issue, waiting for your response.

Thanks.

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team,

Please respond to the comments.

Thanks

**Comment by tiny-ci-servers** [comment]
This issue is stale because it has been open 365 days with no activity. Please comment if you wish to keep this issue open or it will be closed in 30 days.
